### PR TITLE
Chart share link

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: src/components/Share.jsx [hunk 1]
```diff
@@ -40,12 +41,12 @@ const Share = ({
                     ></div>
                     <div className='content'
                         style={{display: ((!loading && !urlGenerated) ? 'flex' : 'none' )}}>
-                        <div
+                        {shortUrlFailed ? <p>Failed to generate link</p> : <div
```
Comment (aminroosta) [inline src/components/Share.jsx]: @shayanbinary `t.translate("Failed to generate link")`

Comment (shayanbinary) [reply src/components/Share.jsx]: @aminroosta ooops, such a shameful mistake . Sorry about that

Comment (aminroosta) [reply src/components/Share.jsx]: No worries, the entire point of code reviews is to catch these simple errors.

**Comment by aminroosta** [comment]
@shayanbinary please update your branch with `dev`.
Let's get this merged 👍

**Pull request merged**